lbrynet/schema/base.py

```diff
@@ -7,18 +6,11 @@
 alphabet = b'123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
 
 
-if six.PY2:
-    iseq, bseq, buffer = (
-        lambda s: map(ord, s),
-        lambda s: ''.join(map(chr, s)),
-        lambda s: s,
-    )
-elif six.PY3:
-    iseq, bseq, buffer = (
-        lambda s: s,
-        bytes,
-        lambda s: s.buffer,
-    )
+iseq, bseq, buffer = (
+    lambda s: s,
+    bytes,
+    lambda s: s.buffer,
+)
```

Comment:
Can you follow this refactor further? For example, there is no benefit to renaming `bytes` to `bseq`, instead it would be better to just use `bytes` where `bseq` currently appears. Same goes for `iseq` (which doesn't even do anything) and `buffer`.

Reply:
To clarify, `iseq` should be dropped altogether.

Reply:
Done